### ISSUE TITLE
Вёрстка на skeeks

### ISSUE THREAD

**Issue by smols** [v3project/app-v3-shop#2]
Добрый день.
подскажите кто добавляет классы к html?
http://joxi.ru/xAeBOQViYV48kr

**Comment by skeeks-semenov** [comment]
Добрый день.
Базовый проект построен на базе http://theme.stepofweb.com/Smarty/v1.2.1/HTML/index.html - этого шаблона.
Этот шаблон содержит в себе различные js и css файлы.
Они же подключается в базовом проекте. И какие то js библиотеки, после построения dom дерева, добавляют на тег html эти классы. 
Какие конкретно не знаю. Можно проверить, например путем поочередного отключения js файлов.

**Comment by skeeks-semenov** [comment]
В любом случае, у вас, как я думаю, будет свой шаблон. В котором не будет этих js файлов, которые добавляют классы к html

**Comment by smols** [comment]
нет. у меня задача на ваш натянуть, да ещё чтобы vendor обновлялся.

**Comment by skeeks-semenov** [comment]
Что натянуть? Откуда задача не обновление vendor?

Проект построен на базе yii2 фреймворка, yii2 фреймворк в свою очередь использует composer. 
Концепция composer именно в том, чтобы обновлять vendor
При углубленном изучении composer и yii2 есть много способов решить задачу или проблему. Поэтому если озвучите почему не нужно обновлять venodor, возможно помогу с решением.
